slices/liblmdb0.yaml

```diff
@@ -0,0 +1,15 @@
+package: liblmdb0
+
+essential:
+  - liblmdb0_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents: 
+      /usr/lib/*-linux-*/liblmdb.so*:
```

Comment:
Can we change this to `/usr/lib/*-linux-*/liblmdb.so.0*` instead of `/usr/lib/*-linux-*/liblmdb.so*`?